plugins/logitech-rallysystem/README.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Plugin: Logitech Rally System — Video Collaboration
+---
+
+## Introduction
+
+This plugin can upgrade the firmware on Logitech Video Collaboration products
+(RallySystem).
+ioctl is used to query system version and serial number from Audio module of RallySystem.
+USB bulk transfer is used to push new firmware image to TableHub module of RallySystem
+
+## Firmware Format
+
+The daemon will decompress the cabinet archive and extract a firmware blob in
+a packed binary file format.
+
+This plugin supports the following protocol ID:
+
+* `com.logitech.vc.rallysystem`
+
+## GUID Generation
+
+These devices use the standard USB DeviceInstanceId values, e.g.
+
+* `USB\VID_046D&DEV_088F`
+
+## Quirk Use
+
+This plugin uses the following plugin-specific quirks:
+
+## Update Behavior
+
+The peripheral firmware is deployed when the device is in normal runtime mode,
+and the device will reset when the new firmware has been written.
+
+## Design Notes
+
+## Vendor ID Security
+
+The vendor ID is set from the USB vendor, in this instance set to `USB:0x046D`
+
+## External Interface Access
+
+This plugin requires read/write access to `/dev/bus/usb`.
+This plugin requires the ioctl interfaces: `HIDIOCGFEATURE`, `HIDIOCSFEATURE`, `HIDIOCGINPUT`.
+
+## Version Considerations
+
+This plugin has been available since fwupd version `1.9.8`.
```

Comment:
@hughsie do you think this should be 1.9.7 or 1.9.8?

Reply:
I guess we could aim for either -- lets leave for 1.9.8 for now.